examples/embed_ipython.py

```diff
@@ -0,0 +1,18 @@
+"""
+Start napari and land directly in an embedded ipython console with qt event loop
+"""
+import napari
+from IPython.terminal.embed import InteractiveShellEmbed
+
+# any code
+text = 'some text'
+
+# initalize viewer
+viewer = napari.Viewer()
+
+# embed ipython and run the magic command to use the qt event loop
+sh = InteractiveShellEmbed()
+sh.run_cell('%gui qt')
```

Comment:
```suggestion
sh.enable_gui("qt")
```

just a weak suggestion... this is a bit more "direct" than calling the magic, which just calls that.  (but the magic is probably more recognizable, so feel free to reject this :)

Reply:
The main reason why I would leave the magic is simply that that's a more generally useful trick to know about :P

Reply:
personally I would vote for:

```python
sh.enable_gui('qt')  # equivalent to using the '%gui qt' magic
```

In Python, do Python things, in a REPL, do REPLy things.

Reply:
and specifically, I like learning about the `enable_gui` API and also not "consuming" a cell. =)

Reply:
Ok, convinced! I just pushed the changes.